encoding/thrift/thriftrw-plugin-yarpc/client.go

```diff
@@ -47,6 +46,7 @@ type Interface interface {
 	<end>
 
 	<range .Service.Functions>
+		<$context := import "context">
```

Comment:
You sure you wanted to move this inside this loop? Could you add a comment explaining why if so?